Pcdmissue140

njaffer opened the pull request:
Used Rubocop gem to auto correct the style issues

mjgiarlo commented:
Can you remove the `vendor/` directory from version control and push that change to this branch? Then we should work on squashing the commits. Happy to help, or @grosscol can help too.

njaffer commented:
I have removed the directory, and tried to squash the commits with Colin's
help. It gave me an error probably because of the commit that I deleted
from Github yesterday. I ran with allow-empty arg. Hope that worked!

On Fri, Aug 14, 2015 at 12:34 PM, Michael J. Giarlo <
notifications@github.com> wrote:

> Can you remove the vendor/ directory from version control and push that
> change to this branch? Then we should work on squashing the commits. Happy
> to help, or @grosscol <https://github.com/grosscol> can help too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra-labs/hydra-pcdm/pull/169#issuecomment-131172272>
> .
>

mjgiarlo commented:
@njaffer Shoot, I don't think it did. Let me see if I can squash them.
